test/rpc_api_mock.go

```diff
@@ -301,3 +354,21 @@ func (mock *mockService) ConsensusPeers(in struct{}, out *[]peer.ID) error {
 	*out = []peer.ID{TestPeerID1, TestPeerID2, TestPeerID3}
 	return nil
 }
+
+// FIXME: dup from util.go
```

Comment:
Not sure if you are planning to skip this in this PR or forgot this comment so pinging you here.

Reply:
planning to skip